sty/smultiling/smultiling.sty.ltxml

```diff
@@ -28,13 +28,13 @@ DefKeyVal('modnl','creators','Semiverbatim');
 DefKeyVal('modnl','contributors','Semiverbatim');
 DefKeyVal('modnl','primary','Semiverbatim');
 DefEnvironment('{modnl} OptionalKeyVals:modnl {}{}',
-         "?#excluded()(<omdoc:theory xml:id='#2.#3'>"
-         .   "?&defined(&GetKeyVal(#1,'creators'))(<dc:creator>&GetKeyVal(#1,'creators')</dc:creator>)()"
-         .   "?&defined(&GetKeyVal(#1,'title'))(<dc:title>&GetKeyVal(#1,'title')</dc:title>)()"
-         .   "?&defined(&GetKeyVal(#1,'contributors'))(<dc:contributor>&GetKeyVal(#1,'contributors')</dc:contributor>)()"
-         .   "<omdoc:imports from='?&GetKeyVal(#1,'load')(&canonical_omdoc_path(&GetKeyVal(#1,'load')))()\##2'/>"
-         .   "#body"
-         . "</omdoc:theory>)",
+          '?#excluded()(<omdoc:theory xml:id="#2.#3">'
+         .   '?&defined(&GetKeyVal(#1,"creator"))(<dc:creator>&GetKeyVal(#1,"creators")</dc:creator>)()'
+         .   '?&defined(&GetKeyVal(#1,"title"))(<dc:title>&GetKeyVal(#1,"title")</dc:title>)()'
+         .   '?&defined(&GetKeyVal(#1,"contributors"))(<dc:contributor>&GetKeyVal(#1,"contributors")</dc:contributor>)()'
+         .   '<omdoc:imports from="?&GetKeyVal(#1,"load")(&canonical_omdoc_path(&GetKeyVal(#1,"load")))()###2"/>'
+         .   '#body'
+         . '</omdoc:theory>)',
```

Comment:
I changed the double quotations marks back and ran a few test cases on GenCS and it seems to be working properly.